rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1017,13 +1017,50 @@ private void logUpdateCollections(Transaction rskTx) {
         eventLogger.logUpdateCollections(sender);
     }
 
-    protected void processSVPFailure(Federation proposedFederation) {
+    protected void updateSvpState(Transaction rskTx) {
+        Optional<Federation> proposedFederationOpt = federationSupport.getProposedFederation();
+        if (proposedFederationOpt.isEmpty()) {
```

Comment:
Maybe add a log here just for good measure.

Reply:
What if we add it at the `DEBUG` level? I think that could be fine at that level.

Reply:
thats a good option, thanks! Will update

Reply:
Updated

Reply:
updated